packages/main/src/components/Page/index.tsx

```diff
@@ -14,21 +14,52 @@ import { CommonProps } from '../../interfaces/CommonProps';
 import styles from './Page.jss';
 
 export interface PagePropTypes extends CommonProps {
+  /**
+   * The title text appearing in the page header bar.
+   */
   title?: string;
+  /**
+   * It is used to set the background color of a page.
+   */
   backgroundDesign?: PageBackgroundDesign;
+  /**
+   * The (optional) custom header of this page. Use this aggregation only when a custom header
```

Comment:
```suggestion
   * The (optional) custom header of this page. Use this prop only when a custom header
```
`aggregation` is in my opinion very tightly coupled to UI5, I would prefer `prop` here is it's more the "react-way"